website/docs/addon_motionbuilder_admin.md

```diff
@@ -10,6 +10,24 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 {versions.MotionBuilder_Badge}
 </ReactMarkdown>
 
-:::note Work in progress
-This part of documentation is still work in progress.
+## Setting your Motion Builder launching version in Application
+
+Users can set up your own Motion Builder version in Application Addon
+You can click **+** button to add new different version.
+
+![AYON Application Motion Builder](assets/motionbuilder_setting_version_application.png)
+
+:::warning Important
+As the integration developed with Motion Builder 2024/2025, some functions may not be working in the version elder than 2024.
 :::
+
+## Publishing Plugins
+There are some validators from Core Addon can be optionally added for Motion Builder.
+
+# Validate Version
+Users can access the validator via `ayon+settings://core/publish/ValidateVersion`
+You can add `motionbuilder` as host names if you want to validate workfile version to make sure the workfile version is in the latest version in AYON.
+
+# Validate Outdated Cotnainer
+Users can access the validator via `ayon+settings://core/publish/ValidateOutdatedContainers`
+You can add `motionbuilder` as host names if you want to validate outdated container to make sure the loaded content is always at the latest version.
```

Comment:
The header sections here seem reversed in indentation?

```suggestion
# Publishing Plugins

There are some validators from Core addon setting that can be optionally enabled for Motion Builder.

## Validate Version
Users can access the validator via `ayon+settings://core/publish/ValidateVersion`
You can add `motionbuilder` as host names if you want to make sure the workfile version is newer than the last published product's version.

## Validate Outdated Containers
Users can access the validator via `ayon+settings://core/publish/ValidateOutdatedContainers`
You can add `motionbuilder` as host names if you want to validate outdated container to make sure the loaded content is always at the latest version.
```

Side note: I really feel like we should fine a way to 'document' these or link to some areas of documentation in "core" or "configuring an application" so we don't need this uniquely documented per application or integration. We should maybe just start of these docs with:

1. To configure the application and executables, see **configuring an application**
2. To enable / disable validation X for specific hosts, see **configuring core validators**

So that we don't need to maintain these duplicate entries. Not needed for this PR - just wanted to mention that some structure design there might be nice. @MustafaJafar ?